rosdep/base.yaml

```diff
@@ -1660,7 +1667,14 @@ libpcl-all-dev:
       packages: [pcl]
   macports: [libpcl]
   opensuse: [pcl]
-  ubuntu: [libpcl-1.7-all-dev]
+  ubuntu:
+    precise: [libpcl-1.7-all-dev]
+    quantal: [libpcl-1.7-all-dev]
+    raring: [libpcl-1.7-all-dev]
+    saucy: [libpcl-1.7-all-dev]
+    trusty: [libpcl-1.7-all-dev]
+    utopic: [libpcl-1.7-all-dev]
+    vivid: [libpcl-dev]
```

Comment:
Utopic should match vivid with libpcl-dev. That release is a straight backport of upstream. I believe the `-dev` package is unversioned since it is not side by side compatible.